oio/cli/election/election.py

```diff
@@ -117,6 +117,12 @@ def take_action(self, parsed_args):
 class ElectionDebug(ElectionCmd):
     """Get debugging information about an election."""
 
+    def get_parser(self, prog_name):
+        parser = super(ElectionDebug, self).get_parser(prog_name)
+        parser.add_argument('--human', action='store_true',
+                            help="Display human-readable dates")
```

Comment:
:+1: Good idea, we should reuse this option

Reply:
Sorry, I moved this to #1904, which is more ZooKeeper themed.